### PR TITLE
fix(Radio): Fix radio dot size

### PR DESCRIPTION
**Pull request by mayank99**
Before:
image

After:
image

## Checklist

- [ ] Add meaningful unit tests for your component (verify that all lines are covered)
- [ ] Verify that all existing tests pass
- [ ] Add component features demo in Storybook (different stories)
- [ ] Approve test images for new stories
- [ ] Add screenshots of the key elements of the component

### CODE REVIEW

File: src/core/Radio/Radio.tsx [hunk 1]
```diff
@@ -80,7 +80,7 @@ export const Radio = React.forwardRef<HTMLInputElement, RadioProps>(
           style={checkmarkStyle}
         >
           <svg viewBox='0 0 16 16' aria-hidden='true' focusable='false'>
-            <circle cx='8' cy='8' r='6' />
+            <circle cx='8' cy='8' r='4' />
```
Comment (veekeys) [inline src/core/Radio/Radio.tsx:83]: Could you please change one image so we have checked state tested and avoid such issues in future?

Comment (veekeys) [reply src/core/Radio/Radio.tsx:83]: same with checkbox, I guess

Comment (mayank99) [reply src/core/Radio/Radio.tsx:83]: I don't have Docker Desktop installed anymore ☹️ Maybe you or @bentleyvk could do it tomorrow during your work hours. If not, I'll reinstall Docker tomorrow

Comment (veekeys) [reply src/core/Radio/Radio.tsx:83]: Done

**Review by veekeys — APPROVED** [review]

**Review by bentleyvk — APPROVED** [review]

**Pull request merged**